PrivacyPreservingTokenizer.ts

```diff
@@ -0,0 +1,87 @@
+
+import { createHash } from 'crypto';
+
+
+export class PrivacyPreservingTokenizer {
+  HASH_FUNCTION: string = 'sha256';
+
+  bloomFilterLength: number;
+  numberOfHashFunctions: number;
+  eta: number; // Probability of *not* flipping a bit
+
+  constructor(bloomFilterLength: number, numberOfHashFunctions: number, privacyBudget: number) {
+    this.bloomFilterLength = bloomFilterLength;
+    this.numberOfHashFunctions = numberOfHashFunctions;
+    this.eta = 1 / (1 + Math.exp(privacyBudget)); // Laplacian?
```

Comment:
https://programming-dp.com/ch3.html - I learnt from this that DP was a Laplacian sampling, but I might be completely wrong

Reply:
Yes, it seems that is using the Laplacian distribution. I have removed the comment, not really needed :p.